docs/csharp/programming-guide/concepts/linq/snippets/set-operators/set-operators.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Add this line to the linq.csproj in the parent folder.

Reply:
I kept it as two projects, moved the one and added a solution. All are targeting .NET 6